lib/deps/clone.js

```diff
@@ -38,6 +38,10 @@ module.exports = function clone(object) {
     return object.toISOString();
   }
 
+  if (object instanceof ArrayBuffer) {
+    return object.slice(0);
+  }
+
```

Comment:
Actually I have a better idea - we should really avoid the additional global `arraybuffer-slice` polyfill. Since we're always calling `slice(0)`, we can just do `return new Uint8Array(object)`, which is the same as `return object.slice(0)`